becquerel/core/peakfinder.py

```diff
@@ -265,6 +265,11 @@ def add_peak(self, xpeak):
             fwhm0 = self.kernel.fwhm(xpeak)
             bw = self.spectrum.bin_widths_raw[0]
             h = int(max(1, 0.2 * fwhm0 / bw))
+
+            # skip peaks that are too close to the edge
+            if (xbin - h < 0) or (xbin + h > len(self.snr) - 1):
+                return
```

Comment:
Can you add a warning that this peak is being skipped?

Reply:
Sure. I added a warning via a print statement. Wasn't quite sure what format you use... let me know if this one is OK.

Reply:
We typically use `warnings.warn()`. There should be an instance of that in the `peakfinder.py` file. In this case we can probably leave the warning type empty (defaults to a `UserWarning` I think) instead of a `DeprecationWarning`.

Reply:
changed it and created a peakfindingwarning